assets/src/util/assignment.js

```diff
@@ -207,13 +207,15 @@ const createAssignmentFields = (
   return sortAssignments(
     assignments.map(a => {
       const {
-        localDate,
+        dueDate,
         pointsPossible,
         assignmentGroupId,
         currentUserSubmission
       } = a
 
-      a.week = calculateWeekOffset(courseStartDate, localDate)
+      a.week = calculateWeekOffset(courseStartDate, dueDate)
+      console.log('dueDate: ' + dueDate)
+      console.log('localDate '+ new Date(dueDate).toLocaleDateString())
```

Comment:
@jaydonkrooss, don't forget to remove these `console.log()` lines as soon as you no longer need them.  Codacy complained about each of them as "Unexpected console statement."  See: https://app.codacy.com/gh/tl-its-umich-edu/my-learning-analytics/pullRequest?bid=39353410&prid=13224791